gateway/Cargo.toml

```diff
@@ -42,6 +42,7 @@ uuid.workspace = true
 omicron-workspace-hack.workspace = true
 oximeter.workspace = true
 oximeter-producer.workspace = true
+oximeter-instruments.workspace = true
```

Comment:
If you want to be a stickler about features, there's an `http-instruments` feature specifically for getting the bit you need for this PR.